streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedTableImpl.java

```diff
@@ -49,11 +49,11 @@
     protected final Serde<V> valSerde;
 
     public KGroupedTableImpl(KStreamBuilder topology,
-                             String name,
+                             String storeName,
                              String sourceName,
                              Serde<K> keySerde,
                              Serde<V> valSerde) {
-        super(topology, name, Collections.singleton(sourceName));
+        super(topology, storeName, Collections.singleton(sourceName));
```

Comment:
I think this is passed in parameter is still the processor name, not the store name right?